Karma/Views/Posts/Details.cshtml

```diff
@@ -115,6 +115,14 @@
         if (text) {
             SendMessage(text);
         }
+        else {
+            Swal.fire({
+                title: 'Could not send a message!',
+                text: 'Your message is empty.',
+                icon: 'error',
+                confirmButtonColor: '#05718f'
+            });
+        }
```

Comment:
There is dedicated pull request with this code change. Be careful merging it, make sure you merge the oldest one first and then the newer one.

Reply:
Will do that.